kyo-prelude/shared/src/main/scala/kyo2/kernel/Frame.scala

```diff
@@ -7,7 +7,7 @@ opaque type Frame <: AnyRef = String
 
 object Frame:
 
-    private val snippetShortMaxChars = 25
```

Comment:
This change makes the frame payload have only the long-form version of the snippet and then the short version is inferred when parsing a frame.